tools/dockerversioning/README.md

```diff
@@ -0,0 +1,70 @@
+# Description
+
```

Comment:
Can we add a comment that this was ported from the runtime-commons library (with link to repo at commit when the code still existed)